### ISSUE TITLE
Resulting reports can NOT be found in /build/reports/  when combined coverage is used in 1.4.0

### ISSUE THREAD

**Issue by TonyTangAndroid** [NeoTech-Software/Android-Root-Coverage-Plugin#37]
It works fine in 1.3.1 per the screenshot. However, when I bump it up to 1.4.0, there is no generated file `./build/reports/jacoco.xml`.

Here is the screenshot taken for 1.3.1. 
<img width="465" alt="1 3 1" src="https://user-images.githubusercontent.com/8612724/117616904-5ab9b080-b19e-11eb-8a12-3ab9966edfda.png">

Here is the screenshot taken for 1.4.0. 

<img width="464" alt="1 4 0" src="https://user-images.githubusercontent.com/8612724/117616939-686f3600-b19e-11eb-97e8-7f315023fe15.png">

**Comment by jarrodrobins** [comment]
I am also having this issue! It's a shame, because it's the main reason I wanted to use this plugin.

**Comment by Rolf-Smit** [comment]
Which Gradle task do you use to generate the report?

**Comment by jarrodrobins** [comment]
I used `./gradlew coverageReport` (minus the colon mentioned in the readme, since that didn't seem to work, as mentioned in #38 ). 

Rolling back to 1.3.1 I used the `./gradlew rootCodeCoverageReport` that I grabbed from the previous version of the readme, which works great. 

If it helps, the only configuration I did was setting the `buildVariant` option as I have a couple of build flavours in my app.

**Comment by Rolf-Smit** [comment]
Yup I just noticed the README.md is incorrect, so I will quickly fix that.
